aat/src/aat/resources/features/F-103 New DefaultValue Column/F-103.feature

```diff
@@ -0,0 +1,23 @@
+@F-103
+Feature: F-103: New DefaultValue Column
```

Comment:
Missing scenario for when there **is** a `DefaultValue`?

Reply:
Fixed in [5e6bcf3](https://github.com/hmcts/ccd-definition-store-api/pull/1111/commits/5e6bcf3530b16e5bff37a8f0c02855a0d686d834)